### DISCUSSION TITLE
Suggestions and Enhancement for Series

### DISCUSSION THREAD

**Discussion by samadtehrani** [rrirower/DP2SceneGraphWizard#113] [General]
1- **Autostart Episodes.** It would be great if when a viewer finishes an episode, the next one plays automatically. 
2- **"Watched" Icon, badge.** Adding a marck or simply "Watched" text on the detail screen for all content, or episode list page, makes a better viewership experience. 
3- **Series on "Continue Watching"**. It would be great to have the series's name and image on the "Continue Watching" row. right now, we see the episode. 
4- **Search and episodes.** I have a fair amount of series, seasons, and episodes, the search option will be much faster if I can limit the search option to series names, not all the content.

**Comment by rrirower** [comment]
Clicking on "Play all" plays all episodes in the series. Is that not what you want?

**Comment by samadtehrani** [comment]
"Play all" isn't quite functioning here. Most series cannot be watched in one session. Imagine a series with one season and 8 episodes, each 45 minutes long. If the viewer quits the show after 2 episodes (or worse, in the middle of the 3rd one), he/she will not be able to continue watching episodes continuously after that. The last episode goes to "continue watching," correct, but when this episode finishes, playback will stop, and there is no way to find which episode he/she was on! I think the best solution is to add an autoplay feature to all episodes. In this case, every time you start an episode, the next one is ready for you, even if you quit watching.

**Reply by rrirower** [reply]
FYI...I've begun looking at integrating these enhancements into the source code.

**Comment by samadtehrani** [comment]
Thank you so much for your help!

**Comment by rrirower** [comment]
I should have this implemented soon.

**Comment by samadtehrani** [comment]
Thank you so much. Looking forward to this one.

**Reply by rrirower** [reply]
See version 1.0.0.4 for these features.
